tests/virt_autotest/install_package.pm

```diff
@@ -52,6 +52,8 @@ sub install_package {
     #install qa_lib_virtauto
     assert_script_run("zypper --non-interactive --gpg-auto-import-keys ref", 180);
     assert_script_run("zypper --non-interactive -n in qa_lib_virtauto",      1800);
+    #install guestfs-tools, which provides additional vm administration utilities
+    assert_script_run("zypper --non-interactive -n in guestfs-tools", 1800);
```

Comment:
After checked guestfs-tools pkg on sles11sp4, sles12sp4,sles15 and sles15sp1 systems, found that guestfs-tools pkg installed automatically as default if $prd_version >=15. So, it'd better to add guestfs-tools pkg as required into libs/qa_lib_virtauto/qa_lib_virtauto.spec file as Alice's mentioned.

Reply:
I recommend you use the helper command `zypper_call` to not repeat the same arguments in all calls

Reply:
This PR may be closed when PR#69 get merged and works.